chatminer/chatparsers.py

```diff
@@ -140,3 +187,4 @@ class StartOfDateType(Enum):
 
 
 DATEREG = r'^((\d{1})|(\d{2}))((\.)|(\/))((\d{1})|(\d{2}))((\.)|(\/))((\d{2}))'
+DATEREG_SIGNAL = r'^\[\d{4}-\d{2}-\d{2} \d{2}:\d{2}\]'
```

Comment:
@gutjuri please refactor these variables into class variables of the respective parser classes